### PR TITLE
feat: Posts

### PR DESCRIPTION
**Pull request by Vishruta-Patil**
- CRUD on Posts
- fix a bug in the profile avatar

### CODE REVIEW

File: src/components/common/modal/EditPostModal.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        >
+          Post
+        </HeroBtn>
+      </div>
+    </div>
+  );
+};
+
+// fixed p-4 left-0 right-0 top-0 h-screen z-50 flex justify-center items-center bg-overlay-color
+// bg-slate-200 rounded-xl absolute top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2 w-8/12  min-w-[60%] p-4
```
Comment (devananth) [inline src/components/common/modal/EditPostModal.tsx:76]: remove it, if not necessary

**Review by devananth — APPROVED** [review]
- Good work, Everything looks clean.

File: src/components/Post/PostCard.tsx [hunk 2]
```diff
@@ -82,6 +91,8 @@ const PostCard = () => {
         )}
       </>
       {/* <CommentCard /> */}
```
Comment (OmKakatkar) [inline src/components/Post/PostCard.tsx:93]: Remove commented code

**Review by OmKakatkar — APPROVED** [review]
Functionality works as expected.
Clean UI, like the card design

Suggestions:
You can close the modal with edit and delete option when the user clicks outside.

**Pull request merged**